File: GrowthWare_VB/GrowthWareFramework/DataAccessLayer/Interfaces/Base/IDBInteraction.vb
Namespace DataAccessLayer.Interfaces.Base
	''' <summary>
	''' IDDBInteraction sets the contract for all
	''' classing inheriting fromm DBInteraction.cs
	''' </summary>
	<System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1709:IdentifiersShouldBeCasedCorrectly", MessageId:="DDB")>
	Public Interface IDDBInteraction

		''' <summary>
		''' Sets or Gets the connection string information.
		''' </summary>
		''' <value>String</value>
		''' <returns>String</returns>
		''' <remarks>Can not be blank</remarks>
		Property ConnectionString As String

	End Interface
End Namespace

File: Net_Core/DataAccess/Interfaces/IDBInteraction.cs
using System;

namespace GrowthWare.DataAccess.Interfaces.Base;
/// <summary>
/// The base interface for Database interaction code
/// </summary>
public interface IDBInteraction
{
    /// <summary>
    /// Gets or sets the connection string.
    /// </summary>
    String ConnectionString { get; set; }
}
